api.yml

```diff
@@ -313,6 +313,66 @@ paths:
               schema:
                 $ref: "#/components/schemas/kmodSignResponse"
 
+  /rpi/keys:
```

Comment:
Is there anything strictly RPi specific? It feels like this only handles RSA keys, maybe we should just call it `/rsa/`?

Reply:
yes, in hindsight that's a good idea. I didn't really know if there would be something RPI specific when I started so I choose the safe bet.